api/models/node.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type NodeStatus struct {
-	Primary bool `json:"primary" form:"primary" binding:"Required"`
+	Primary string `json:"mode" form:"mode" binding:"Required"`
```

Comment:
sorry i missed changing that back after i changed them to mode. good that you saw it